test/block.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { describe } from 'mocha';
+import * as fixtures from './fixtures/block_deserialize.json';
+import * as block from '../ts_src/block';
+
+describe('block deserialization ', () => {
+  fixtures.test.forEach(f => {
+    block.Block.fromBuffer(Buffer.from(f.hex, 'hex'));
```

Comment:
you need to check result is non null? @louisinger

Reply:
this checks if `fromBuffer` throws an error so LGTM. Best way to test is to check if all the block's fields are deserialized but may be hard to make the fixtures

Reply:
I mean we need an `it` ro specify the test

ie.

```suggestion
  it('should de-serialize dyna-fed blcoks', () => {

  });
```